### PR TITLE
Translation blog 2018-08-02-dynamically-expand-volume-csi-translation

### PR DESCRIPTION
**Pull request by gaoguangze111**
Blog 2018-08-02-dynamically-expand-volume-csi-translation.md translation.
Translation task : https://github.com/k8smeetup/website-tasks/issues/3335
https://github.com/k8smeetup/website-tasks/issues/3335

### CODE REVIEW

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+---
+layout: blog
+title:  'Dynamically Expand Volume with CSI and Kubernetes'
+date:   2018-08-02
+---
+-->
+
+---
+layout: blog
+title:  '使用CSI和Kubernetes实现卷的动态扩容'
```
Comment (ydcool) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:11]:
```diff
- title:  '使用CSI和Kubernetes实现卷的动态扩容'
+ title:  '使用 CSI 和 Kubernetes 实现卷的动态扩容'
```
中英文中间添加空格分隔

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+* awsElasticBlockStore
+* OpenStack Cinder
+* glusterfs
+* rbd
+
+<!--
+In order to enable this feature, we should set feature gate `ExpandPersistentVolumes` true and turn on the `PersistentVolumeClaimResize` admission plugin. Once `PersistentVolumeClaimResize` has been enabled, resizing will be allowed by a Storage Class whose `allowVolumeExpansion` field is set to true.
+-->
+
+为了启用此功能，我们应该将功能门 `ExpandPersistentVolumes` 设置为 true 并打开 `PersistentVolumeClaimResize` 准入插件。 一旦启用了 `PersistentVolumeClaimResize`，则其对应的 `allowVolumeExpansion` 字段设置为 true 的存储类将允许调整大小。
```
Comment (ydcool) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:73]: `feature gate` 通常译作 “特性开关”

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+
+![csi diagram](/images/blog/2018-08-02-dynamically-expand-volume-csi/csi-diagram.png)
+
+<!--
+* Three new external components are introduced to decouple Kubernetes and Storage Provider logic
+* Blue arrows present the conventional way to call against API Server
+* Red arrows present gRPC to call against Volume Driver
+-->
+
+* 引入了三个新的外部组件以分离 Kubernetes 和存储提供程序逻辑
```
Comment (ydcool) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:121]: `decouple` => 解耦

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+For more details, please visit: https://github.com/container-storage-interface/spec/blob/master/spec.md
+-->
+
+更多详细信息，请访问：https://github.com/container-storage-interface/spec/blob/master/spec.md
+
+<!--
+## Extend CSI and Kubernetes
+-->
+
+##扩展 CSI 和 Kubernetes
```
Comment (ydcool) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:135]:
```diff
- ##扩展 CSI 和 Kubernetes
+ ## 扩展 CSI 和 Kubernetes
```

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+In order to enable the feature of expanding volume atop Kubernetes, we should extend several components including CSI specification, “in-tree” volume plugin, external-provisioner and external-attacher.
+-->
+
+为了实现在 Kubernetes 上扩展卷的功能，我们应该扩展几个组件，包括 CSI 规范，“in-tree” 卷插件，external-provisioner 和 external-attacher。
+
+<!--
+## Extend CSI spec
+-->
+
+##扩展CSI规范
```
Comment (ydcool) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:147]:
```diff
- ##扩展CSI规范
+ ## 扩展CSI规范
```

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+ rpc NodeResizeVolume (NodeResizeVolumeRequest)
+   returns (NodeResizeVolumeResponse) {}
+}
+```
+
+<!--
+## Extend “In-Tree” Volume Plugin
+-->
+
+##扩展 “In-Tree” 卷插件
```
Comment (ydcool) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:179]:
```diff
- ##扩展 “In-Tree” 卷插件
+ ## 扩展 “In-Tree” 卷插件
```

File: content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+title:  'Dynamically Expand Volume with CSI and Kubernetes'
+date:   2018-08-02
+---
+-->
+
+---
+layout: blog
+title:  '使用 CSI 和 Kubernetes 实现卷的动态扩容'
+date:   2018-08-02
+---
```
Comment (tengqm) [inline content/zh/blog/_posts/2018-08-02-dynamically-expand-volume-csi.md:13]: pls move line 9-13 to the head of the file.

**Comment by tengqm** [comment]
/lgtm
/approve
Thanks for the fix.

**Pull request merged**